drivers/docker/handle.go

```diff
@@ -308,6 +308,9 @@ func (h *taskHandle) run() {
 		h.logger.Error("failed to wait for container; already terminated")
 	}
 
+	ctx, cancel := context.WithTimeout(context.Background(), dockerTimeout)
```

Comment:
This 5min timeout seems excessive, but we didn't have a context for this or the `ContainerStop` requests before so I'm not sure what we want to do here. Inspect seems like it should return basically immediately?